examples/raytracing-rint/scene.txt

```diff
@@ -0,0 +1 @@
+https://www.desmos.com/3d/linjbopc4y
```

Comment:
I've read that they are permalinks (https://help.desmos.com/hc/en-us/articles/4405901719309-Saving-and-Managing-Graphs). However, I will include scene data just in case.

Reply:
Ok, I added the variables and equations needed to visualize the scene in Desmos 3-D. I believe there were only two `scene.txt` files.